header.php

```diff
@@ -23,7 +23,13 @@
 
 <body <?php body_class(); ?>>
 
-<?php wp_body_open(); ?>
+<?php
+if ( function_exists( 'wp_body_open' ) ) {
```

Comment:
Could we remove this commit please? I've added compatability for wp_body_open here:
https://github.com/BinaryMoon/jarvis/blob/master/inc/compat.php

Reply:
I've made the requested change with the updated PR.